Userland/Utilities/shot.cpp

```diff
@@ -120,7 +120,7 @@ int main(int argc, char** argv)
         app->exec();
 
         crop_region = container.region();
-        if (crop_region.is_empty()) {
+        if (crop_region.has_value()) {
```

Comment:
Since `container.region()` always returns an `IntRect` `crop_region` will always have a value and thus output cancelled and stop.
Also even if `container.region()` returned an optional should this not be `!crop_region.has_value()`? Since only if we have a region do we continue?

Again I could have the logical completely flipped but than perhaps more clear names should be used to indicate that they are negatives.

Reply:
How does it look now? `crop_region` will always have value and I need to check if it is empty.

Reply:
That seems good.
Quickly looking at the code perhaps: `SelectableLayover::region()` should return an `Optional` but you can leave that for a future PR. Since this already addresses the issue.